### ISSUE TITLE
Autobuilder Support for C++ with Gradle

### ISSUE THREAD

**Issue by JLLeitschuh** [github/codeql#5376]
**Description of the issue**

Gradle can be used to compile/test native C++ code. It would be nice if the autobuilder supported executing Gradle to build C/C++ code.

Demo/POC can be seen here:

https://github.com/gradle/native-platform/blob/master/.lgtm.yml
https://lgtm.com/query/5826053983844977996/
https://lgtm.com/projects/g/gradle/native-platform/alerts/?mode=list

**Comment by jbj** [comment]
Thanks for pointing this out. I'll take this as a data point, and I'll keep watching for other requests to support Gradle.

**Comment by JLLeitschuh** [comment]
Should this stay open?

**Comment by jbj** [comment]
Yes, let's keep this issue open. The auto-close bot is new, and we're going to change it so it'll be less eager.
